oppijanumerorekisteri-domain/src/main/java/fi/vm/sade/oppijanumerorekisteri/repositories/impl/HenkiloRepositoryImpl.java

```diff
@@ -144,4 +148,60 @@
         });
         return henkiloDtoList;
     }
+
+    @Override
+    public List<HenkiloViiteDto> findHenkiloViitteesByHenkilo(HenkiloCriteria query) {
+        Where where = buildSqlConditions(query, "h");
+        SQLQuery q = em.unwrap(Session.class)
```

Comment:
Nyt kun tosiaan selvisi se, että tuohon henkilön muokkausajankohtaan ei oikein voida luottaa (eli sitä ei päivitetä kun tehdään pelkästään henkilöiden viitteisiin muutoksia, joilla ei ole omaa aikaleimaansa), niin varmaan tosiaan luotettavin ratkaisu olisi hakea ne kaikki. Tällöin toki joukkoa voisi hieman rajata jos tulisi tuo oid-lista tällä tavoin POST-pyyntönä. Tämä voisi olla selkein ratkaisu ja tosiaan saadaan nopeammaksikin kun hierarkiaa ei tarvitse tukea.